pkg/manifest/manifest.go

```diff
@@ -114,11 +116,62 @@ func (m *Manifest) UnmarshalJSON(in []byte) error {
 	return validateResourceId(m.id)
 }
 
+const featureSetAnnotation = "release.openshift.io/feature-set"
+
+var knownFeatureSets = sets.String{}
+
+func init() {
```

Comment:
Globals and the `init` package-singleton are also something I'd expect to find at the top of the file.  But :shrug: search is cheap, so if feels better to you to keep them here, I don't mind.